tests/Unit/Controllers/Document/DocumentAccessHelperTest.php

```
@@ -0,0 +1,129 @@
+    }
+
+    #[Test]
+    public function helper_method_correctly_identifies_ownership_via_creator()
+    {
+        $task = factory(Task::class)->create([
+            'user_created_id' => $this->owner->id,
+            'user_assigned_id' => $this->otherUser->id,
+            'client_id' => $this->client->id,
+        ]);
+
+        $document = factory(Document::class)->create([
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$document'. (undefined)

(UnusedLocalVariable)